internal/controllers/topology/cluster/desired_state_test.go

```diff
@@ -549,273 +552,660 @@ func TestComputeControlPlane(t *testing.T) {
 }
 
 func TestComputeControlPlaneVersion(t *testing.T) {
```

Comment:
Very nice and readable table tests overall. Great work!

Reply:
Thank you! :)